m2e-maven-runtime/org.eclipse.m2e.maven.indexer/src/main/resources/META-INF/sisu/javax.inject.Named

```diff
@@ -0,0 +1,20 @@
+org.apache.maven.index.creator.MavenArchetypeArtifactInfoIndexCreator
```

Comment:
Can you please add a not in the class and/or in the commit message about why this class is now necessary?
Couldn't it be taken from somewhere else?

Reply:
Actually I discovered I could adequately generate this file so I added some comments about it to the pom.xml